packages/flutter_driver/lib/src/driver.dart

```diff
@@ -15,13 +15,37 @@ import 'health.dart';
 import 'message.dart';
 import 'timeline.dart';
 
+enum TracingCategory {
```

Comment:
They're not streams, they're stream identifiers, but I can't bring myself to call them TimelineStreamIdentifiers :-).  I'll rename TracingCategory to TimelineStream along with code to produce the startup-time histogram json in the follow-up PR.